presto-main/src/main/java/com/facebook/presto/execution/TaskManager.java

```diff
@@ -43,6 +43,11 @@
     TaskInfo getTaskInfo(TaskId taskId);
 
     /**
+     * Extracts the status for the specified task from the info.
```

Comment:
I think we should document this simply as `Gets the status for the specified task`.  Then in the future we can make the status independent of info.